src/layer/tile/TileLayer.WMS.js

```diff
@@ -110,9 +110,11 @@ export const TileLayerWMS = TileLayer.extend({
 		    [min.y, min.x, max.y, max.x] :
 		    [min.x, min.y, max.x, max.y]).join(','),
 		    url = TileLayer.prototype.getTileUrl.call(this, coords);
-		return url +
-			getParamString(this.wmsParams, url, this.options.uppercase) +
-			(this.options.uppercase ? '&BBOX=' : '&bbox=') + bbox;
+		const url0 = new URL(url);

```

Comment:
I think there is no need to keep `url`. Let us replace `url0` with `url`
```suggestion
		const url = new URL(TileLayer.prototype.getTileUrl.call(this, coords));
```

Reply:
Thanks, that's clearer.